EmbedSnap.php

```diff
@@ -60,7 +60,8 @@ function renderEmbedSnap ($input, $argv, $parser) {
 		{
 			$height = $argv['height'];
 		}
-		if (!empty($project, $user)) {
+		if (!empty($project)) {
+			if (!empty($project)) {
```

Comment:
Should not this be "$user" instead?
```suggestion
			if (!empty($user)) {
```